docs/examples/haproxy/config.toml

```diff
@@ -3,8 +3,9 @@ dockerURL = "unix:///var/run/docker.sock"
 
 [[extensions]]
 name = "haproxy"
-configPath = "/usr/local/etc/haproxy.cfg"
-pidPath = "/etc/haproxy.pid"
+configPath = "/usr/local/etc/haproxy/haproxy.cfg"
```

Comment:
Ya. [haproxy.org isn't too helpful](https://www.google.com/webhp?sourceid=chrome-instant&ion=1&espv=2&ie=UTF-8#q=site:www.haproxy.org+%22haproxy.cfg%22). I think it mostly comes down where distros package it, seems like [Ubuntu](https://www.google.com/webhp?sourceid=chrome-instant&ion=1&espv=2&ie=UTF-8#q=ubuntu%20haproxy.cfg) and [CentOS](https://www.google.com/webhp?sourceid=chrome-instant&ion=1&espv=2&ie=UTF-8#q=centos%20haproxy.cfg) favour `/etc/haproxy/haproxy.cfg`.

That's always been my personal preference too (e.g. /etc/appname/appname.conf, /etc/appname/appname.yml, etc.). But honestly I don't feel strongly about it here. I'm fine changing it back to `/usr/local/etc/haproxy.cfg` if that's your preference.